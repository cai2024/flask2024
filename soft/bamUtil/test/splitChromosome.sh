../bin/bam  splitChromosome --in testFilesLibBam/sortedBam.bam --out results/splitSortedBam --noph 2> results/splitChromosome.txt \
&& diff results/splitChromosome.txt expected/splitChromosome.txt \
&& diff results/splitSortedBam1.bam expected/splitSortedBam1.bam \
&& diff results/splitSortedBam2.bam expected/splitSortedBam2.bam \
&& diff results/splitSortedBam3.bam expected/splitSortedBam3.bam \
&& diff results/splitSortedBamUnknownChrom.bam expected/splitSortedBamUnknownChrom.bam \

if [ $? -ne 0 ]
then
  exit 1
fi
