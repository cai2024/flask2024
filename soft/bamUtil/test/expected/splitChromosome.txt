Reference Name: 1 has 5 records
Reference Name: 2 has 2 records
Reference Name: 3 has 1 records
Reference Name: * has 2 records
Number of records = 10
Returning: 0 (SUCCESS)
